src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WithdrawableService.kt

```diff
@@ -16,24 +16,25 @@ class WithdrawableService(
   @Lazy private val placementRequestService: PlacementRequestService,
   @Lazy private val bookingService: BookingService,
   @Lazy private val placementApplicationService: PlacementApplicationService,
+  @Lazy private val userAccessService: UserAccessService,
 ) {
   val approvedPremisesWithdrawnByPPBookingWithdrawnReasonId: UUID =
     UUID.fromString("d39572ea-9e42-460c-ae88-b6b30fca0b09")
 
-  fun allWithdrawables(application: ApprovedPremisesApplicationEntity): Withdrawables {
+  fun allWithdrawables(application: ApprovedPremisesApplicationEntity, showAllBookings: Boolean = false): Withdrawables {
```

Comment:
could we maybe flip this boolean and call it something like 'onlyShowUserManageableBookings'? (or something a bit more concise :) )